api/android/api/src/main/java/org/nnsuite/nnstreamer/Pipeline.java

```diff
@@ -157,6 +158,23 @@ public Pipeline(@NonNull String description, @Nullable StateChangeCallback callb
         mStateCallback = callback;
     }
 
+    /**
+     * Checks the element is registered and available on the pipeline.
+     *
+     * @param element The name of GStreamer element
+     *
+     * @return true if the element is available
+     *
+     * @throws IllegalArgumentException if given param is invalid
+     */
+    public static boolean isElementAvailable(@NonNull String element) {
+        if (element == null || element.isEmpty()) {
```

Comment:
I think this null check is not necessary as `element` is annotated as `@NonNull`

Reply:
NonNull annotation is just warning message in android-studio. This does not guarantee the passed param is not null.

Reply:
Yes.. it's the caller's responsibility.